src/errors/Messages.js

```diff
@@ -4,12 +4,12 @@ const { register } = require('./DJSError');
 
 const Messages = {
   CLIENT_INVALID_OPTION: (prop, must) => `The ${prop} option must be ${must}`,
+  CLIENT_INVALID_PROVIDED_SHARDS: 'None of the shards you provided were valid.',
```

Comment:
`None of the provided shards were valid.` would be more consistent ~~most of~~ the other error messages.